controllers/orderController.js

```diff
@@ -0,0 +1,27 @@
+const orderService = require("../services/orderService");
+const { catchAsync } = require("../utils/error");
+
+const getOrderUser = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+  const { cartId, productId, quantity } = req.body;
+  console.log(
+    cartId,
+    "--------------------------------------------------------"
+  );
+  const result = await orderService.getOrderUser(
+    userId,
+    cartId,
+    productId,
+    quantity
+  );
+  return res.status(200).json(result);
+});
+
+const postOrder = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+  const { totalPrice, cartInfos } = req.body;
+  orderService.postOrder(userId, totalPrice, cartInfos);
```

Comment:
```suggestion
  await orderService.postOrder(userId, totalPrice, cartInfos);
```
await가 없으면 실제로 postOrder가 성공하지 않았는데 response가 먼저갈수도 있습니다.

Reply:
수정했습니다 !